### ISSUE TITLE
[BUG] App grid fade in/out transition speed if animation is disabled

### ISSUE THREAD

**Issue by meybonomme** [G-dH/vertical-workspaces#171]
I really like the option to disable app grid animation because it's nice, clean and distraction free. Although the name says there are no animations, there is a fade in/out animation which is very nice.

The transition from the app grid to the desktop is nice and smooth, but the transition from the desktop to the app grid feels too quick and eye distracting. It would be nice if the transition from the desktop to the app grid was smoother as well.

https://github.com/G-dH/vertical-workspaces/assets/147217565/a6a5133c-3439-4e5d-9cc4-0ef2966476c5

**Comment by G-dH** [comment]
The transition has the same duration, it's an optical illusion. Slowing down the beginning of the transition would help.

**Comment by meybonomme** [comment]
> Slowing down the beginning of the transition would help.

It seems to me that you've already made this change. The fade-in animation looks much nicer now.

**Comment by G-dH** [comment]
Sounds good, but I didn’t change anything. The transition between states is handled by the default overview animation mode in GNOME Shell.

**Comment by meybonomme** [comment]
It seems that a change has been made in the GNOME Shell because for some time now, I haven't felt the need to adjust the animation. I also watched a screen recording I made earlier, and compared to that, the current animation has a much smoother transition and is more eye-friendly.

**Comment by G-dH** [comment]
Same wallpaper and animation speed?

**Comment by meybonomme** [comment]
Could this also be the reason why the animation seems smoother in this case? I noticed this change quite some time ago; it's been weeks already.

Anyway, everything is fine now, and I don’t see the need to change anything else.

**Comment by G-dH** [comment]
Yes, that’s the reason. The delay helps prevent the animation from skipping its initial frames. More windows require more time to generate their clones for the overview, and the default Shell may end up skipping the animation entirely. This change is not new.

**Comment by meybonomme** [comment]
Thanks to this, all V-Shell animations are much smoother in various situations than before. Thank you, and I'm closing this issue.
